features/doctrine/date_filter.feature

```diff
@@ -656,6 +656,15 @@ Feature: Date filter on collections
 
   @dropSchema
   @createSchema
+  Scenario: Get collection filtered by date that is not a datetime
+    Given there is "30" dummydate objects with dummyDate
+    When I send a "GET" request to "/dummy_dates?dummyDate[after]=2015-04-28"
+    Then the response status code should be 200
```

Comment:
You don't test anything in here if you don't test the content of the response. Therefore, I don't understand what is this test trying to prove

Reply:
If I run this test on my database without the patch it fails to status code `500`. This is just to ensure a non-regression.

Reply:
Does it affects MySQL or Postgres? it it is the case, maybe should we also run the Behat test suite with such DBMS.

Reply:
As long as they have two types for dates (ie date vs datetime) yes. And yes I think we should.

I came across this when using a `date` on an index (it's weird to keep time on an index imo).